charts/fluent-bit/templates/vpa.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if and (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1") (.Values.verticalpodautoscaling.enabled) }}
```

Comment:
```suggestion
{{- if and (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1/VerticalPodAutoscaler") .Values.verticalpodautoscaling.enabled }}
```

This check won't currently detect the VPA CRD and is also not guaranteed to work in certain patterns such as `helm template` so I generally recommend avoiding this pattern. However it is currently in use this chart so the following change should make it work correctly.